chainer/variable.py

```diff
@@ -730,6 +739,31 @@ def to_gpu(self, device=None):
             if node._data is not None:
                 node.retain_data()
 
+    def to_intel64(self):
+        """Copies the data and gradient arrays to intel64 specific mdarray
```

Comment:
A nitpick but should we add a punctation to this sentence?

Reply:
Yes exactly, a small detail.

Reply:
Fixed. Thanks!